atlas/src/views/SearchView.tsx

```diff
@@ -21,7 +21,7 @@ const VideosHeader = styled.h5`
 `
 const SearchView: React.FC<SearchViewProps> = ({ search = '' }) => {
   const [selectedIndex, setSelectedIndex] = useState(0)
-  const { data, loading } = useQuery<Search, SearchVariables>(SEARCH, { variables: { query_string: search } })
+  const { data, loading, error } = useQuery<Search, SearchVariables>(SEARCH, { variables: { query_string: search } })
 
   const getChannelsAndVideos = (loading: boolean, data: Search | undefined) => {
```

Comment:
I know this is old code but why not have this below loading and error handlers? Then we don't need to handle loading case there at all